presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/ResourceGroupSpec.java

```diff
@@ -145,4 +147,57 @@ public ResourceGroupNameTemplate getName()
     {
         return hardCpuLimit;
     }
+
+    @Override
+    public boolean equals(Object other)
+    {
+        if (other == this) {
+            return true;
+        }
+        if (!(other instanceof ResourceGroupSpec)) {
+            return false;
+        }
+        ResourceGroupSpec that = (ResourceGroupSpec) other;
+        // Subgroups is not included
```

Comment:
To determine whether a currently loaded subgroup has a different config than the db. Changed the method and added subgroups back to equals.